src/main/kotlin/study/step3/Racing.kt

```diff
@@ -0,0 +1,28 @@
+package study.step3
+
+import study.step3.inputview.InputView
+import study.step3.resultview.ResultView
+
+class Racing(
+    private val inputView: InputView,
+    private val resultView: ResultView
+) {
+
+    fun start() {
+        val (numOfCar, numOfAttempt) = inputView.receive()
+        runStages(numOfAttempt, createCars(numOfCar))
+    }
+
+    private fun runStages(totalRounds: Long, cars: List<Car>) {
+        println("실행 결과")
+
+        (0 until totalRounds).forEach { _ ->
+            cars.forEach { it.move() }
+            resultView.printResult(cars)
+        }
+    }
+
+    private fun createCars(numOfCar: Long): List<Car> {
+        return (0 until numOfCar).map { Car(RandomMovementConditionGenerator()) }
+    }
```

Comment:
이렇게 하나의 표현식으로 나타낼 수 있다면 다음과 같이 코드를 작성할 수도 있어요🙂
```suggestion
private fun createCars(numOfCar: Long): List<Car> = List(numOfCar.toInt()) { Car(RandomMovementConditionGenerator())}
```